src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -291,8 +291,8 @@ def request(self, url, params, headers=None, verb='GET',
                 data = self.parse_body(bbuf.getvalue(), decode)
         else:
             data = bbuf.getvalue()
-            msg = 'url=%s, code=%s, reason=%s, headers=%s' \
-                  % (url, header.status, header.reason, header.header)
+            msg = 'url=%s, code=%s, reason=%s, headers=%s, result=%s' \
```

Comment:
+1

Reply:
perfect! i had the fear that you did not want to print `data` because we risk to fill some logs in case of some weird errors, but i was stuck on a failing unittest for 1h before realizing that just printing that would have given me the stacktrace I needed.

Reply:
If it fails, I believe amount of data will be fairly small. So, I think it's a good tradeoff. But definitely something to keep in mind in case things go wild :-D